code/modules/mob/living/carbon/xenomorph/Facehuggers.dm

```diff
@@ -192,19 +195,19 @@
 		Die()
 		return FALSE
 
-	lifecycle -= 2 SECONDS
+	lifecycle -= 4 SECONDS
 	return TRUE
 
 /obj/item/clothing/mask/facehugger/proc/check_neighbours()
-	var/obj/item/clothing/mask/facehugger/F
-	var/count = 0
-	for(F in get_turf(src))
-		if(F.stat == CONSCIOUS && !F.sterile)
-			count++
-		if(count > 2) //Was 5, our rules got much tighter
-			visible_message("<span class='xenowarning'>The facehugger is furiously cannibalized by the nearby horde of other ones!</span>")
-			qdel(src)
-			return TRUE
+	if(isturf(loc))
```

Comment:
```suggestion
	if(isturf(loc))
```
early return

Reply:
It is still won't grant a TRUE return value even if the location happen to be a turf, plus it's not too deeply indented, hence why I didn't do the early return.